guardian/test/slack/test_utils/extensions/validate_section_block_extension.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:guardian/slack/model/slack_section_block.dart';
+
+import 'validate_text_object_extension.dart';
+
+extension ValidateSectionBlockExtension on SlackSectionBlock {
```

Comment:
@vladlaba it would be great if we can discuss this use case in details when you are available